rmd/Chargen Parts.json

```diff
@@ -661,243 +661,243 @@
+    "Enabled": true
   },
   "440145": {
     "OriginalText": "ラミア・ヘッド破",
-    "Text": "",
-    "Enabled": false
+    "Text": "Lamia Head (Damaged)",
+    "Enabled": true
   },
   "440146": {
     "OriginalText": "グリース・ヘッド破",
-    "Text": "",
-    "Enabled": false
+    "Text": "Griss Head (Damaged)",
+    "Enabled": true
   },
   "440147": {
     "OriginalText": "ケルビナ・ヘッド破",
-    "Text": "",
-    "Enabled": false
+    "Text": "Cherubina Head (Damaged)",
+    "Enabled": true
   },
   "440148": {
     "OriginalText": "メディウム・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Medium Head",
+    "Enabled": true
   },
   "440149": {
     "OriginalText": "ヘルマ・ヘッド破",
-    "Text": "",
-    "Enabled": false
+    "Text": "Helma Head (Damaged)",
+    "Enabled": true
   },
   "440150": {
     "OriginalText": "ゴルドファム・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Gold Fam Head",
+    "Enabled": true
   },
   "440151": {
     "OriginalText": "ガラテ・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Garate Head",
+    "Enabled": true
   },
   "440152": {
     "OriginalText": "アルキ・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Alki Head",
+    "Enabled": true
   },
   "440153": {
     "OriginalText": "ラビーソ・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Rabiso Head",
+    "Enabled": true
   },
   "440154": {
     "OriginalText": "マーヴ・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Marv Head",
+    "Enabled": true
   },
   "440155": {
     "OriginalText": "ノヴァファム・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Nova Fam Head",
+    "Enabled": true
   },
   "440156": {
     "OriginalText": "シャロン・ヘッド",
-    "Text": "",
-    "Enabled": false
+    "Text": "Sharon Head",
+    "Enabled": true
   },
   "440157": {
     "OriginalText": "ベリーショート",
-    "Text": "",
-    "Enabled": false
+    "Text": "Very Short",
+    "Enabled": true
   },
   "440158": {
     "OriginalText": "ショートマッシュ",
-    "Text": "",
-    "Enabled": false
+    "Text": "Short Mash",
+    "Enabled": true
   },
   "440159": {
     "OriginalText": "スパイキーウルフ",
-    "Text": "",
-    "Enabled": false
+    "Text": "Spiky Wolf",
+    "Enabled": true
   },
   "440160": {
     "OriginalText": "カジュアル",
-    "Text": "",
-    "Enabled": false
+    "Text": "Casual",
+    "Enabled": true
   },
   "440161": {
     "OriginalText": "ショートウルフ",
-    "Text": "",
-    "Enabled": false
+    "Text": "Short Wolf",
+    "Enabled": true
   },
   "440162": {
     "OriginalText": "ミディアムレイヤー",
-    "Text": "",
-    "Enabled": false
+    "Text": "Medium Layer",
+    "Enabled": true
   },
   "440163": {
     "OriginalText": "ブレイズ",
-    "Text": "",
-    "Enabled": false
+    "Text": "Blaze",
+    "Enabled": true
   },
   "440164": {
     "OriginalText": "クラシックショート",
-    "Text": "",
-    "Enabled": false
+    "Text": "Classic Short",
+    "Enabled": true
   },
   "440165": {
     "OriginalText": "スパイク",
-    "Text": "",
-    "Enabled": false
+    "Text": "Spike,
```

Comment:
Where the `"`?

Reply:
Gone, and...accidentally forgotten. Sorry about that!